wiki/Tournaments/Official_support/en.md

```diff
@@ -2,168 +2,171 @@
 
 The [osu! team](/wiki/People/The_Team) runs a program where they may opt to provide community-run tournaments that abide by a certain set of criteria and expectations with extra support, at their discretion.
 
+This article was last updated on January 5, 2022. For any queries and clarifications, please consult the `#tournaments` channel on the [osu!dev Discord server](https://discord.com/invite/ppy) or send an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
+
 ## Benefits
 
 Officially supported tournaments have access to:
 
-- **Profile badge prizes for first-place winners**
-- Assessment for a main-menu banner during major matches (generally quarterfinals and above), subject to availability and criteria (see the [relevant section](#requesting-main-menu-banner-support) below)
-- A listing in the [automatic tournament registrations system](https://osu.ppy.sh/community/tournaments)
-- A dedicated screening process to prevent players with serious integrity infringements (cheating, multiaccounting, tournament bans) from playing
+- A dedicated [screening process](#tournament-screening) to prevent players with serious integrity infringements (cheating, multi-accounting, tournament bans) from playing.
+- [Profile badge prizes](#profile-badges) for first-place winners.
+- Assessment for a [main-menu banner advertisement](#requesting-main-menu-banner-support) during major matches, subject to availability and criteria.
```

Comment:
```suggestion
- Potential consideration for a [main-menu banner advertisement](#requesting-main-menu-banner-support) during major matches
```

I don't think we should list this as something explicitly "available" if that makes sense, since only a very narrow band of tournaments will ever really qualify for it. The current wording leans towards the 'this is a given' side of things and it really isn't, especially when we're so particular about what goes on the main menu.